build.gradle

```diff
@@ -831,3 +844,12 @@ tasks.withType(Test) {
         exceptionFormat 'full'
     }
 }
+
+integrationTest {
+    testLogging {
+        events "passed", "skipped", "failed"
+        showStackTraces = true
+        showStandardStreams = true
+        //exceptionFormat = TestExceptionFormat.FULL
+    }
+}
```

Comment:
Does this look right?

Reply:
https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.logging.TestLogging.html

- showStackTraces seems to be enabled by default
- you can use the `TestLogEvent` enum in the events section if you don't want to hardcode strings

I found a fancier one but it's probably more than we need at this point:
https://stackoverflow.com/questions/3963708/gradle-how-to-display-test-results-in-the-console-in-real-time